tests/test_tagging.py

```diff
@@ -0,0 +1,31 @@
+import struct
+import unittest
+
+from rsocket.exceptions import RSocketError
+from rsocket.extensions.tagging import TaggingMetadata
+from rsocket.extensions.mimetypes import WellKnownMimeTypes
+
+
+class TestTaggingMetadata(unittest.TestCase):
+    def test_serialize_max_length(self):
```

Comment:
Thank you for the bug fix / pull request.

If you please, remove the unittest subclass and preferably write the tests in a way similar to the rest of the tests. a good place (and example) for these tests would be tests/rsocket/test_extensions.py.

As in the examples in those files the data to parse should be explicitly written into the test and not constructed using methods.

Reply:
Hi, I changed the unit test